xsnap/sources/xsnap-worker.c

```diff
@@ -208,6 +208,20 @@ static char *renderTimestamps() {
 	return timestampBuffer;
 }
 
+int data_pointer_value = 1234;
+int bss_pointer_value = 0;
+
+void print_pointers() {
+	char *heap_pointer = malloc(1);
```

Comment:
It's only by reviewing the whole file for calls to `print_pointers()` that I can tell that not freeing this is OK because the program exits immediately. Consider freeing it before returning to reduce coupling.

Reply:
aye, good point